prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -299,6 +308,42 @@ func popRandom(set sets.String) string {
 	return sel
 }
 
+func popActiveReviewer(ghc githubClient, log *logrus.Entry, set sets.String) string {
+	for {
+		if set.Len() == 0 {
+			break
+		}
+		candidate := popRandom(set)
+		busy, err := isUserBusy(ghc, candidate)
+		if err != nil {
+			log.Errorf("error checking user availability: %v", err)
+		}
+		if !busy {
+			return candidate
+		}
+	}
+	return ""
+}
+
+type githubAvailabilityQuery struct {
+	User struct {
+		Login  githubql.String
+		Status struct {
+			IndicatesLimitedAvailability githubql.Boolean
+		}
+	} `graphql:"user(login: $user)"`
+}
+
+func isUserBusy(ghc githubClient, user string) (bool, error) {
```

Comment:
can we record the output of this so we don't double check in one run?

Reply:
Sure.. like having a set like `busyUsers` that we populate with logins that we know for certain are busy, and have `isUserBusy` check that first before a call?